src/doc/reference.md

```diff
@@ -311,6 +311,19 @@ A _string literal_ is a sequence of any Unicode characters enclosed within two
 which must be _escaped_ by a preceding `U+005C` character (`\`), or a _raw
 string literal_.
 
+A multi-line string literal may be defined by terminating each line with a
+`U+005C` character (`\`) immediately before the newline. This causes the
+`U+005C` character, the newline, and all whitespace at the beginning of the
+next line to be ignored.
+
+```rust
+let a = "foobar";
+let b = "foo\
+         bar";
+```
+
+`a` is equivalent to `b`.
```

Comment:
Instead of this line, could you add an `assert_eq!(a,b)` to the code above? It makes it more implicit, and besides will get tested by our doctests.